src/components/Dropdowns/AccessibleDropdown.tsx

```diff
@@ -9,8 +9,14 @@ type AccessibleDropdownProps = {
   /** Text label to include with the dropdown. If null or undefined, hides the label. */
   label?: string | null;
 
-  /** Contents to be rendered inside the dropdown. */
-  dropdownContent: ReactElement;
+  /**
+   * Contents to be rendered inside the dropdown. Can either be a React element or a function
+   * that returns a React element.
+   *
+   * @param setOpenState: A callback that can be used to force the dropdown open or closed.
+   * Useful for closing the dropdown after a selection has been made.
+   */
+  dropdownContent: ReactElement | ((setOpenState: (open: boolean) => void) => ReactElement);
```

Comment:
Something hard to parse about this...  dropdownContent can take a function that returns a react element, but the function takes as its argument another function that is a setter based on whether the dropdown is open or closed?  I'd consider factoring that out as a special type just for readability.

Reply:
Fixed!